cpp/src/Slice/Grammar.y

```diff
@@ -151,54 +142,57 @@ namespace
 // Specify a custom location type for storing the location & filename of tokens.
 %define api.location.type {Slice::TokenContext}
 
+// Tell Bison to emit additional information with it's syntax error messages (like which tokens it expected).
+%define parse.error detailed
+
 // All keyword tokens. Make sure to modify the "keyword" rule in this
 // file if the list of keywords is changed. Also make sure to add the
 // keyword to the keyword table in Scanner.l.
```

Comment:
These strings are the "human readable" token names Bison will use when reporting errors.